File: Sduty_Server/src/main/java/com/d108/sduty/dto/Timeline.java
package com.d108.sduty.dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Setter
@Getter
@NoArgsConstructor
@AllArgsConstructor
public class Timeline {
	private Profile profile ;
	private Story story ;
	private int cntReply; 
	private List<Reply> replies;
	private boolean likes;
	private boolean scrap;
	private String jobHashtag;
	private List<String> subjectHashtag;
	
	
}

File: Sduty_Admin/app/src/main/java/com/d108/sduty_admin/model/dto/Timeline.kt
package com.d108.sduty.sduty_admin.dto

import android.provider.ContactsContract
import com.d108.sduty_admin.model.dto.InterestHashtag
import com.d108.sduty_admin.model.dto.JobHashtag
import com.d108.sduty_admin.model.dto.Reply
import com.d108.sduty_admin.model.dto.Story

data class Timeline(var profile: ContactsContract.Profile, var story: Story, var cntReply: Int, var replies: MutableList<Reply>, var numLikes: Int, var likes: Boolean, var scrap: Boolean, var jobHashtag: JobHashtag, var interestHashtags: List<InterestHashtag>) {
}